docs/source/getting_started/installation.md

```diff
@@ -0,0 +1,61 @@
+## Installation
+Elyra can be installed via PyPi:
+
+### Prerequisites
+* [NodeJS 12+](https://nodejs.org/en/)
+* [Python 3.X](https://www.anaconda.com/distribution/)
+
+##### Optional
+* [Anaconda](https://www.anaconda.com/distribution/) 
+* [Docker](https://docs.docker.com/install//) - If using the docker image
+
+#### JupyterLab support
+
+* [JupyterLab](https://github.com/jupyterlab/jupyterlab) 1.x is supported on **Elyra 0.10.0 and below**
+* [JupyterLab](https://github.com/jupyterlab/jupyterlab) 2.x is supported on **Elyra 0.11.0-rc0 and above**
+
+via PyPi:
+```bash
+pip install elyra && jupyter lab build
+```
+Note: Ubuntu and CentOS users may need to use `pip3 install elyra` 
+
+### Verify Installation 
+```bash
+jupyter serverextension list
+```
+Should output:
+```
+config dir: /usr/local/etc/jupyter
+    elyra  enabled
+    - Validating...
+      elyra  OK
+    jupyterlab  enabled
+    - Validating...
+      jupyterlab 2.1.1 OK
+    jupyterlab_git  enabled
+    - Validating...
+      jupyterlab_git 0.20.0rc0 OK
+    nbdime  enabled
+    - Validating...
+      nbdime 2.0.0 OK
+
+```
+```bash
+jupyter labextension list
+```
+Should output:
+```
+Known labextensions:
+   app dir: /Users/lresende/opt/anaconda/envs/dev/share/jupyter/lab
+        @elyra/application v0.11.0-dev  enabled  OK*
```

Comment:
Let's use the latest release version here (v0.10.1) rather than a dev build.  Same for the other elyra versions below.  We'll probably create a script that search/replaces elyra versions that runs as part of the release build.

Reply:
Fixed.